src/trossen_rslidar_node.cpp

```diff
@@ -0,0 +1,280 @@
+/*********************************************************************************************************************
+Copyright (c) 2020 RoboSense
```

Comment:
For example, if we wanted to ad ourselves to this file (which I believe we are able to)

```suggestion
Copyright (c) 2020 RoboSense
Copyright (c) 2024 Trossen Robotics
```